extension/src/popup/views/UnlockAccount.tsx

```diff
@@ -1,30 +1,45 @@
 import React from "react";
 import get from "lodash/get";
-import { useDispatch } from "react-redux";
+import { useDispatch, useSelector } from "react-redux";
 import { useLocation } from "react-router-dom";
 import styled from "styled-components";
+import { Formik } from "formik";
 
-import { POPUP_WIDTH } from "constants/dimensions";
 import { ROUTES } from "popup/constants/routes";
-import { COLOR_PALETTE } from "popup/constants/styles";
+import { COLOR_PALETTE, FONT_WEIGHT } from "popup/constants/styles";
 
 import { navigateTo, openTab } from "popup/helpers/navigate";
 import { newTabHref } from "helpers/urls";
 
 import { BasicButton } from "popup/basics/Buttons";
+import {
+  Form,
+  SubmitButton,
+  FormRow,
+  ApiErrorMessage,
+  TextField,
+} from "popup/basics/Forms";
+import { SubviewWrapper } from "popup/basics/AccountSubview";
 
 import { Header } from "popup/components/Header";
-import { PasswordConfirmation } from "popup/components/PasswordConfirmation";
```

Comment:
restoring the old version of this file bc it is no longer sharing any code with the new subviews